typings/mkdirp.d.ts

```diff
@@ -0,0 +1,9 @@
+import * as fs from "fs";
+declare namespace MkDirp {
+    interface mkdirp {
+        (path: string): Promise<string[]>;
+    }
+}
+
+declare var mkdirp: MkDirp.mkdirp;
+export = mkdirp;
```

Comment:
You have `"@types/mkdirp": "^0.5.1",` in `pagkace.json`, which means you shouldn't need this file?  In fact, I suspect it conflicts and only one of the two is being used (I'm not sure which).

Reply:
Alright, I'll remove `"@types/mkdirp": "^0.5.1"`